app/components/candidate_interface/application_status_tag_component.rb

```diff
@@ -35,6 +35,12 @@ def colour
       end
     end
 
+    def days_since_submission(application_form)
+      return if application_form.submitted_at.nil?
+
+      (Time.zone.now.to_date - application_form.submitted_at.to_date).to_i
```

Comment:
we need to use the `sent_to_provider_at` field on an `ApplicationChoice`. This is because we're only going to use submitted_at for the candidates first application, and then we don't write to it again, so we need this to be on an application choice level

Reply:
I think we should consider this also being a method on the model level. As I think it makes sense for `ApplicationStatusTagComponent` to only be aware of the status of the app

Reply:
Both good shouts, thanks! 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/8573/commits/7fa7182b26ca94df627dc73fbb66e4b117c49425